src/esp/gfx/TextureVisualizerShader.cpp

```diff
@@ -99,9 +101,10 @@ TextureVisualizerShader::TextureVisualizerShader(Flags flags) : flags_(flags) {
   // set default offset, scale based on flags
   if (flags_ & Flag::DepthTexture) {
     setColorMapTransformation(1.0f / 512.0f, 1.0f / 1000.0f);
+  } else if (flags & Flag::ObjectIdTexture) {
+    setColorMapTransformation(1.0f / 512.0f,
```

Comment:
@mosra:
I was thinking the wrapping of the colorMapTexture found [here](https://github.com/facebookresearch/habitat-sim/pull/1207/files#diff-1f42624b53785fbfad33558bea811592548000ef11dedc0533d69086ed35db23R95).
I believe for depth rendering it should be set to `ClampToEdge` while for semantic rendering it should be set to `Repeat`. How do you think?

Reply:
@mosra:
Another question: we can clear the depth, clear the color, but how can we clear the objectId (another color attachment)

Reply:
use the `clearDepth()` / `clearColor()` APIs on the framebuffer, instead of `clear()`, those allow you to clear also non-integer attachments